### PR TITLE
RAMP-757: Add geoip2 vmod to query maxmind db

### PR DESCRIPTION
**Pull request by paupm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,25 @@ FROM varnish:${VERSION}
...
- rm -rf /var/lib/apt/lists/*
+RUN set -e; \
+    apt-get update; \
+    apt-get -y install prometheus-varnish-exporter libmaxminddb-dev $VMOD_DEPS /pkgs/*.deb;
+
+# Install geoip2 https://github.com/varnishcache-friends/libvmod-geoip2
+RUN mkdir /tmp/module_to_build && \
+    curl -fLo /tmp/module_to_build/src.tar.gz https://github.com/varnishcache-friends/libvmod-geoip2/archive/refs/tags/v1.3.0.tar.gz
+
+WORKDIR /tmp/module_to_build
```
Comment (mindhells) [inline Dockerfile]: not sure changing the workdir is a good idea

File: Dockerfile [hunk 2]
```diff
@@ -4,7 +4,25 @@ FROM varnish:${VERSION}
 
 USER root
 
-RUN apt-get update && apt-get install -y prometheus-varnish-exporter &&\
- rm -rf /var/lib/apt/lists/*
+RUN set -e; \
+    apt-get update; \
+    apt-get -y install prometheus-varnish-exporter libmaxminddb-dev $VMOD_DEPS /pkgs/*.deb;
```
Comment (mindhells) [inline Dockerfile:9]: libmaxminddb-dev... is there a -prod version 😅 ?

File: Dockerfile (hunk 1 above)
Comment (paupm) [reply Dockerfile]: probably it is not, I'm going to replace it by `cd`

File: Dockerfile (hunk 2 above)
Comment (paupm) [reply Dockerfile:9]: No 😄 it's a requirement to build [libvmod-geoip2](https://github.com/varnishcache-friends/libvmod-geoip2)

**Review by mindhells — APPROVED** [review]

**Pull request merged**